### ISSUE TITLE
Restoring data crashes app

### ISSUE THREAD

**Issue by joshua50n** [goodtohear/habits#81]

**Comment by joshua50n** [comment]
Went from iPhone X to iPhone 13 pro

**Comment by michaelforrest** [comment]
Hmm, that’s not good. Do you have the json file?

**Comment by joshua50n** [comment]
Yes. I click the link and then it closes out of the app

**Comment by michaelforrest** [comment]
Yes but there should also be a json file attachment that might work better (you need to drag it in via computer though). If you can see a json attachment, can you tell me the file size? How long have you been using the app?

**Comment by joshua50n** [comment]
where do I drag it to from the comp?

size is 754kb

I’ve been using the app for several years now.. maybe 5 years?

**Comment by michaelforrest** [comment]
Yeah okay that's a lot of data and I think the link will fail to encode it! If you are able to connect to a computer, you should be able to drop the json file onto your phone as per the instructions here: https://goodtohear.co.uk/blog/post/How_to_restore_your_Good_Habits_data

**Comment by joshua50n** [comment]
is there a way to do this without the finder sync? I have an older OS and would like not to upgrade.

**Comment by michaelforrest** [comment]
There is not another way in the current version of Habits. 

I can see a way to do it, but it will take some development time. 

Would you be interested in Beta testing a new build when it's ready? If you share your Apple ID email in an email to michael@goodtohear.co.uk I can add you.
